modules/mono/glue/GodotSharp/GodotSharp/Core/NativeInterop/ExceptionUtils.cs

```diff
@@ -171,6 +171,10 @@ private unsafe static string GetCallErrorWhere(godot_string_name method, godot_v
                 if (argCount >= 1)
                 {
                     methodstr = VariantUtils.ConvertToString(*args[0]);
+                    if (error.Error == godot_variant_call_error_error.GODOT_CALL_ERROR_CALL_ERROR_INVALID_ARGUMENT)
+                    {
+                        error.Argument += 1;
```

Comment:
I guess it is not pragmatically possible to be on a `GODOT_CALL_ERROR_CALL_ERROR_INVALID_ARGUMENT` with `argCount == 1` and go out of range here?

Reply:
This is also only if the method is named `call` or it's `TreeItem::call_recursive`, so I think it's pretty safe. It should also match the [GDScript implementation](https://github.com/godotengine/godot/blob/96a386f3c424af96d950ee5098b4b0e4907c9508/modules/gdscript/gdscript_vm.cpp#L1787-L1789).